sql/src/test/java/io/crate/testing/SQLExecutor.java

```diff
@@ -570,6 +582,28 @@ public static Builder builder(ClusterService clusterService, int numNodes, Rando
         return new Builder(clusterService, numNodes, random);
     }
 
+    public static DocTableInfo tableInfo(RelationName name, String stmt, ClusterService clusterService) {
+        try {
+            SQLExecutor executor = builder(clusterService)
+                .addTable(stmt)
+                .build();
```

Comment:
The builder creation and build altogether is fairly expensive, I wonder if we should mark these methods somehow to indicate that if more than 1 table is required it is recommended to create the executor only once?

E.g in T3 there are 2 loops that use this method inside the loops.

Reply:
I've added method documentation and also removed the T3 usages.